src/_styles/_mobileMenu.scss

```diff
@@ -0,0 +1,69 @@
+.mobile-menu-button {
+  color: $dark-blue;

```

Comment:
Let's add `cursor: pointer` to button "Menu" and "Close".

Reply:
- [ ] Done